src/domains/nodo-app/04_apim_wisp_dismantling_policies.tf

```diff
@@ -25,22 +25,6 @@ resource "azurerm_api_management_named_value" "wisp_channel_whitelist_named_valu
   value               = var.wisp_converter.channel_whitelist
 }
 
-resource "azurerm_api_management_named_value" "wisp_station_whitelist_named_value" {
```

Comment:
should be kept until the final transition to the prod environment, no?

Reply:
Yes, I added this important consideration in the description of the PR